packages/store/tests/dispatch.spec.ts

```diff
@@ -57,6 +57,28 @@ describe('Dispatch', () => {
     expect(observedCalls).toEqual(['handleError(...)', 'observer.error(...)']);
   }));
 
+  it('should run outside zone and return back in zone', async(() => {
+    @State<number>({
+      name: 'counter',
+      defaults: 0
+    })
+    class MyState {
+      @Action(Increment)
+      increment() {
+        expect(NgZone.isInAngularZone()).toBe(false);
+      }
+    }
+
+    TestBed.configureTestingModule({
+      imports: [NgxsModule.forRoot([MyState]), NgxsModule.forFeature([])]
+    });
+
+    const store: Store = TestBed.get(Store);
```

Comment:
I think that we should run the whole test in the Angular zone and test that we are in the zone before we call `dispatch` just to be explicit. The test will then demonstrate that we go from the angular zone, out of it, and back into the zone.